regression/contracts/assigns_null_pointer_02/test.desc

```diff
@@ -0,0 +1,13 @@
+CORE
+main.c
+--enforce-contract foo
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
```

Comment:
Could you also add a test where we have an allocation that non-deterministically fails and pass the pointer to a function with an assigns clause including this pointer? I want to make sure we cover cases where the pointer comes from mallocs.

Reply:
I have added a test case and marked it as a `KNOWNBUG`.